docs-src/ref-programs-consensus.scrbl

```diff
@@ -115,8 +115,7 @@ A @tech{continue statement} may be written without the preceding identifier upda
  [] = [];
  continue; }
 
-A @tech{continue statement} must be @tech{dominated} by a @tech{consensus transfer}, which means that the body of a @tech{while statement} must always @reachin{commit();} before calling @reachin{continue;}.
-This restriction may be lifted in future versions of Reach, which will perform termination checking.
+A @tech{continue statement} must perform a @tech{consensus transfer} before calling @reachin{continue;}.
```

Comment:
The original sentences are good.

The thing that your commit does is it makes so that a continue statement, despite being listed in the consensus step section, can actually occur in a step, provided the RHS modifies the state into a consensus step.